python/core/auto_generated/qgsconditionalstyle.sip.in

```diff
@@ -43,12 +43,24 @@ Each row will check be checked against each rule.
 .. seealso:: :py:func:`rowStyles`
 
 .. versionadded:: 2.12
+%End
+
+    QgsConditionalStyle constraintFailureStyles( QgsFieldConstraints::ConstraintStrength strength );
+%Docstring
+Returns a style associated to a constraint failure.
+
+:param strength: the type of constraint
+
+.. versionadded:: 3.30
 %End
 
     void setFieldStyles( const QString &fieldName, const QList<QgsConditionalStyle> &styles );
 %Docstring
 Set the conditional ``styles`` for a field, with the specified ``fieldName``.
 
+The field value is inserted into a 'value' variable to conduct
+expression checks.
```

Comment:
this is outdated, right?

Reply:
@nyalldawson , it's actually not no, it expands the documentation to explain what @value means when setting field styles' expression.